### PR TITLE
Consistent variable names throughout the project

### PR DESCRIPTION
**Pull request by vhouchouas**

### CODE REVIEW

File: CalDAVClient.php [hunk 1]
```diff
@@ -121,7 +121,7 @@ function updateEvents($urls) {
     }
 
     // get event etags from the server
-    function getetags() {
+    function getETags() {
```
Comment (gturri) [inline CalDAVClient.php:124]: IIUC this renaming should also be applied on `tests/CalDAVTest.php`

File: CalDAVClient.php [hunk 2]
```diff
@@ -174,7 +174,7 @@ function getetags() {
     
     // get the ctag of the calendar on the server
     // @see https://sabre.io/dav/building-a-caldav-client/
-    function getctag() {
+    function getCTag() {
```
Comment (gturri) [inline CalDAVClient.php:177]: IIUC this renaming should also be performed on:
- `FSAgenda.php` (L126, and similarly on L135)
- `tests/CalDAVTest.php`
- `tests/FilesystemCacheTest.php`

File: FSAgenda.php [hunk 3]
```diff
@@ -276,25 +276,25 @@ public function updateAttendee(string $url, string $usermail, bool $add, ?string
             }
         }
         
-        $new_etag = $this->caldav_client->updateEvent($url, $etag, $vcal->serialize());
-        if($new_etag === false) {
+        $new_ETag = $this->caldav_client->updateEvent($vCalendarFilename, $ETag, $vCalendar->serialize());
```
Comment (gturri) [inline FSAgenda.php:279]: It's a detail, but for consistency you might also want to apply a similar renaming on `tests/CalDAVTest.php` (L57, 58)

File: localcache.php [hunk 4]
```diff
@@ -9,14 +9,14 @@
...
-    public function getctag();
-    public function setctag($ctag);
-    public function eventExists($eventName);
-    public function getEventEtag($eventName);
-    public function deleteEvent($eventName);
-    public function getAllEventsNames();
-    public function addEvent($eventName, $eventData, $etag);
+    public function getSerializedEvent($vCalendarFilename);
+    public function getCTag();
+    public function setCTag($CTag);
```
Comment (gturri) [inline localcache.php:14]: this reaming should also be performed on 
- `FSAgenda.php`
- `tests/FilesystemCacheTest.php`

File: localcache.php [hunk 5]
```diff
@@ -9,14 +9,14 @@
...
+    public function eventExists($vCalendarFilename);
+    public function getEventETag($vCalendarFilename);
```
Comment (gturri) [inline localcache.php:16]: This renaming should also be performed on `tests/FilesystemCacheTest.php` (L48)

Comment (gturri) [inline localcache.php:16]: This renaming should also be performed on `tests/FilesystemCacheTest.php`

File: localcache.php [hunk 6]
```diff
@@ -9,14 +9,14 @@
...
+    public function deleteEvent($vCalendarFilename);
+    public function getAllEventsFilenames();
```
Comment (gturri) [inline localcache.php:18]: This renaming should also be performed on `tests/FilesystemCacheTest.php`

**Comment by vhouchouas** [comment]
Thanks for having pointing the remaining misnamed variables/methods.
I have `push --force` in order put everything in the same commit.
Tests are OK.

**Pull request merged**